src/node_jsvmapi.h

```diff
@@ -18,6 +18,7 @@
 #ifndef SRC_NODE_JSVMAPI_H_
 #define SRC_NODE_JSVMAPI_H_
 
+#include "node.h"
```

Comment:
Having said that I think that all modules will have to include node.h as they need the node_module definition which it contains.